src/Sulu/Bundle/AdminBundle/Exception/DeletionImpossibleChildPermissionsException.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Exception;
+
+class DeletionImpossibleChildPermissionsException extends \Exception
+{
+    public const EXCEPTION_CODE = 12346; // TODO change
```

Comment:
Is there a convention on what exception codes should be used?

Reply:
Every bundle had its prefix number in the path example media was `5` and so 5000-5999 area was reserved for media exception codes thats the only one I remember.